src/Hng.Application/Services/JobListingService.cs

```diff
@@ -0,0 +1,28 @@
+using AutoMapper;
+using Hng.Application.Dto;
+using Hng.Application.Interfaces;
+using Hng.Domain.Entities;
+using Hng.Infrastructure.Repository.Interface;
+
+namespace Hng.Application.Services
+{
```

Comment:
Move all services to infrastructure

Reply:
moving all services to the infrastructure layer would go against the principles of clean architecture and separation of concerns. i think we should maintain the distinction between application and infrastructure services @NdubuisiJr 

Or how would you recommend we keep things modular? tried moving this to Infrastructure and had dep issues.

Reply:
If we were following a strict architecture, you guys would have been writing CQRS and event sourcing. I want all services in infrastructure exactly because of separation of concerns. Currently that’s where others are, yours won’t go anywhere else.